clients/src/main/java/org/apache/kafka/common/protocol/types/Type.java

```diff
@@ -132,9 +159,14 @@ public Byte validate(Object item) {
             else
                 throw new SchemaException(item + " is not a Byte.");
         }
+
+        @Override
+        public String documentation() {
+            return "Represents an integer between -2<sup>7</sup> and 2<sup>7</sup>-1 inclusive.";
```

Comment:
even clearer: "Represents a signed integer"

Reply:
I can't find a good way to fit 'signed' in the sentence. I can not simply replace specifying the boundaries, because then it might create the impression that we are talking about 4 byte integers. Having 'signed' and the between clause next to each other seems to be redundant.
Do you agree?